pkgs/by-name/kd/kdocker/package.nix

```diff
@@ -2,49 +2,36 @@
   stdenv,
   lib,
   fetchFromGitHub,
-  qmake,
-  wrapQtAppsHook,
+  cmake,
   libX11,
-  libXmu,
-  libXpm,
-  qtbase,
-  qtx11extras,
+  libxcb,
+  perl, # For pod2man
+  qt6,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "kdocker";
-  version = "5.4";
+  version = "6.2";
```

Comment:
@r-ryantm may be able to automatically generate update PRs (and notify you) if you add an [update script](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#automatic-package-updates)